clusterctl/cmd/create_cluster.go

```diff
@@ -163,6 +163,9 @@ func getProvider(provider string) (clusterdeployer.ProviderDeployer, error) {
 	case "azure":
 		//Work being done at https://github.com/platform9/azure-provider
 		return nil, errors.New("Azure not yet implemented")
+	case "openstack":
+		//Work being done at https://github.com/kubernetes-sigs/cluster-api-provider-openstack
+		return nil, errors.New("OpenStack not yet implemented")
```

Comment:
Would it be better to print an error message with a link to a README in the openstack repo that explains how to create a cluster?

Reply:
Let's hold this. I think proceeding with #360 is a more tenable path to unblock you (and other out of tree providers).